frontend/js/components/bottomPlayerInfo.js

```diff
@@ -0,0 +1,54 @@
+var React = require('react');
```

Comment:
Perhaps merge "bottomPlayerInfo.js" and "topPlayerInfo.js" into just one "playerInfo.js"? Although their displays are reversed, they do share most of their code, including the hardcoded 150 for setInterval which is much better if it only occurs in one place, and I can hardly imagine any scenario where you would add a new item/display/piece of logic to one and not the other.